pkg/tracker/index.go

```diff
@@ -151,14 +156,17 @@ func BuildIndexTrackers(cfg *config.Config, db db.DB) ([]Tracker, error) {
 type IndexObject struct {
 	URL      string `json:"URL"`
 	JSONPath string `json:"JSONPath"`
+	Interval uint   `json:"interval"`
 }
+
 type IndexTracker struct {
 	DB         db.DB
 	Name       string
 	Identifier string
 	Symbols    []string
 	Source     DataSource
 	JSONPath   string
+	Interval   uint //seconds minimum that we wait before pulling again
```

Comment:
Please follow Golang commentary guidelines.
https://golang.org/doc/effective_go.html#commentary

Here and in few other places